src/test/scala/com/yotpo/metorikku/metric/stepActions/dataQuality/HasUniquenessTest.scala

```diff
@@ -0,0 +1,70 @@
+package com.yotpo.metorikku.metric.stepActions.dataQuality
+
+import com.amazon.deequ.checks.Check
+import com.yotpo.metorikku.metric.stepActions.dataQuality.operators.HasUniqueness
+import org.apache.log4j.{Level, Logger}
+import org.apache.spark.sql.SparkSession
+import org.scalatest.{BeforeAndAfterEach, FunSuite}
+
+class HasUniquenessTest extends FunSuite with BeforeAndAfterEach {
+  private var sparkSession : SparkSession = _
+  Logger.getLogger("org").setLevel(Level.WARN)
+
+  override def beforeEach() {
+    sparkSession = SparkSession.builder().appName("dq tests")
+      .master("local")
+      .config("", "")
+      .getOrCreate()
+  }
+
+  private def valideHasUniquenessOverDf(employeeData: Seq[(String, Int, String, Int, Int)], columns: Seq[String],
+                                        assertion: Double => Boolean, level: String) = {
+    val sqlContext = sparkSession.sqlContext
+    val hasUniquenessCheck = new HasUniqueness(level = Some(level),columns = columns, assertion=assertion)
+    val dqCheckDefinitionList = DataQualityCheckList(List[DataQualityCheck](DataQualityCheck(None, None, hasUniqueness = Some(hasUniquenessCheck))), None)
+    import sqlContext.implicits._
+
+    val dfName = "employee_data"
+    val df = employeeData.toDF(dfName, "id", "name", "fake", "fake2")
+    df.createOrReplaceTempView(dfName)
+
+    dqCheckDefinitionList.runChecks(dfName)
+  }
+
+  test("has_uniqueness on a non-unique field with level error should raise exception") {
+    val employeeData = Seq(
+      ("Maria", 1, "Smith", 111, 1111),
+      ("Josh", 1, "Smith", 222, 2222)
+    )
+    val level = "error"
+
+    val thrown = intercept[Exception] {
+      valideHasUniquenessOverDf(employeeData, Seq("id", "name"), Check.IsOne, level)
```

Comment:
Can you define the function? it should be more clear this function is (double => boolean)